generate_index.go

```diff
@@ -11,6 +11,7 @@ func generate_index(w io.Writer, domain string, r []repository) error {
 <html>
 <head>
 <meta charset="utf-8">
+<title>{{$.Domain}} Go Modules</title>
```

Comment:
Is the $ necessary?

Reply:
Oh I was just blindly copying the <h2>. It's not necessary in either places, so I now removed it.